crates/swc_ecma_utils/src/lib.rs

```diff
@@ -979,7 +979,14 @@ pub trait ExprExt {
         match *expr {
             Expr::Lit(ref l) => match *l {
                 Lit::Str(Str { ref value, .. }) => Known(Cow::Borrowed(value)),
-                Lit::Num(ref n) => Known(format!("{}", n).into()),
+                Lit::Num(ref n) => {
+                    if n.value == -0.0 {
+                        return Known(Cow::Borrowed("0"));
+                    }
+
+                    let mut buffer = ryu::Buffer::new();
+                    Known(Cow::Owned(buffer.format(n.value).to_string()))
```

Comment:
@magic-akari It seems like `ryu` returns `"1.0"` for 1. Should we revert this change?

Reply:
Let's try `ryu-js`